notebook/services/contents/tests/test_manager.py

```diff
@@ -108,7 +108,7 @@ def test_checkpoint_subdir(self):
         self.assertEqual(cp_dir, os.path.join(root, cpm.checkpoint_dir, cp_name))
         self.assertEqual(cp_subdir, os.path.join(root, subd, cpm.checkpoint_dir, cp_name))
 
-    @dec.skip_win32
+    @dec.skipif(sys.platform == 'win32' and sys.version_info[0] < 3)
```

Comment:
Ah, nice. I guess our build servers have got the 'creators update' and therefore are able to create symlinks. :-)